packages/lisk-p2p/src/response_handler.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { valid as isValidVersion } from 'semver';
+import { isAlpha, isIP, isNumeric, isPort } from 'validator';
+import { InvalidPeer } from './errors';
+import { PeerConfig } from './peer';
+
+const IPV4_NUMBER = 4;
+const IPV6_NUMBER = 6;
+
+interface RawPeerObject {
+	readonly height?: string;
+	readonly ip: string;
+	readonly os?: string;
+	readonly version?: string;
+	readonly wsPort: string;
+}
+
+export const checkForValidPeerAddress = (
```

Comment:
check and valid is similar meaning.
maybe `validPeerAddress` or `validatePeerAddress`?